pkgs/tools/networking/wiretrustee/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "wiretrustee";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "wiretrustee";
+    repo = "wiretrustee";
+    rev = "v${version}";
+    sha256 = "051c234rvjjkadjhwxs7x3lph40pidvkx7si1cvsrlcnyjrcddcy";
+  };
+
+  vendorSha256 = "1sz5lbw0cvzspg19n45x8kx5xhs26hf2a726ls1byc6r5cg52hcw";
+
+  doCheck = false;
```

Comment:
Why is this disabled? Please add a comment.

Reply:
I'll look into it, when I'm back on my feet.